Doc/library/functools.rst

```diff
@@ -358,6 +358,18 @@ The :mod:`functools` module defines the following functions:
       >>> basetwo('10010')
       18
 
+   If ``Placeholder`` sentinels are present in *args*, they will be filled first
```

Comment:
```suggestion
   If :class:`Placeholder` sentinels are present in *args*, they will be filled first
```

Reply:
As double back-ticked wrap is used for None in docs, I think this is not applicable anymore.